CODEOWNERS

```diff
@@ -1,8 +1,41 @@
+# By default, require admin approval for files that do not match any other rules
+* @mosaicml/composer-team-admins
+
 # Require admin approval to modify all files in the root of the repository
+# This includes setup.py, the README, and the CODEOWNERS file itself!
 /* @mosaicml/composer-team-admins
 
 # Require admin approval to change the CI build configuration
+# All CI Changes should be reviewed for security
```

Comment:
Let's add me explicitly here as well for now.

Reply:
Done!